taskcluster/kinds/inference/kind.yml

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - translations_taskgraph.transforms.skip_unless_inference_changed
+    - translations_taskgraph.transforms.worker_selection
+    - taskgraph.transforms.task_context
+    - taskgraph.transforms.run:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+
+task-defaults:
+  task-context:
+    from-parameters:
+      base_rev: base_rev
+    substitution-fields:
+      - run.command
+    worker:
+  run:
+    using: run-task
+    cwd: '{checkout}'
+  worker-type: b-cpu
+  worker:
+    max-run-time: 3600
+    docker-image: {in-tree: inference}
+    # 128 happens when cloning this repository fails
+    retry-exit-status: [128]
+
+tasks:
+  build-local:
+    description: "Build the inference engine locally"
+    run-on-tasks-for: ["github-push", "github-pull-request"]
+    run:
+      command:
+        - bash
+        - -c
+        - >-
+            task inference-build
+
+  test-local:
+    description: "Run local-build tests for the inference engine"
+    worker-type: b-cpu
```

Comment:
It looks like you dropped `worker-type` and some other things from `build-local` but not `test-local` nor `build-wasm`. (I assume this was an oversight!)

Reply:
Yes, just an oversight, thanks for catching that! I fixed it now.